.github/workflows/cibuildwheel.yml

```diff
@@ -46,6 +46,7 @@ jobs:
           python -c "import cftime; print(f'cftime v{cftime.__version__}')" &&
           python -m pip install check-manifest cython pytest pytest-cov &&
           python -m pytest -vv {package}/test
+        CIBW_BUILD_FRONTEND: 'pip; args: --pre --extra-index-url "https://pypi.anaconda.org/scientific-python-nightly-wheels/simple"'
```

Comment:
This part (or something like it) is necessary if you want to release before 2.0 actually lands

Reply:
https://pypi.org/project/numpy/2.0.0rc1/#files

Reply:
Ohhh yes you're right I forgot it's on PyPI proper now :facepalm:

Reply:
Thank you for being proactive and for helping with this. I am also looking forward to a numpy 2 compatible release. Just trying to save you time.